### DISCUSSION TITLE
Smiths Gloves (i)

### DISCUSSION THREAD

**Discussion by PinkyOSRS** [runelite/runelite#16762] [Ideas]
Hi

I guess it isn't much of an idea but I didn't know where else to post this.

the smiths gloves (i) work as replacement for ice gloves during the heroes quest for those who have gotten the ice gloves before doing the quest. the plugin tells me to go kill the ice queen eventhough I already have all the items to finish the quest.

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.

**Reply by PinkyOSRS** [reply]
ok ty
